docker-compose.yml

```diff
@@ -19,3 +41,8 @@ vault:
     - consul:consul
   volumes:
     - vault/assets/conf.d:/vault/conf.d
+  environment:
+    - "SERVICE_NAME=vault"
+    - "SERVICE_TAGS=vault"
+    - "VAULT_ADDR=http://0.0.0.0:8200"
+    - "VAULT_TOKEN=lofasz"
```

Comment:
docker-compose.yml needs some love it's a bit outdated. I'll throw a quick
fix in for that.

2015-10-27 10:58 GMT+01:00 Peter-Paul <notifications@github.com>:

> In docker-compose.yml
> <https://github.com/Sweebr/terra-con-vault/pull/1#discussion_r43101276>:
>
> > @@ -19,3 +41,8 @@ vault:
> >      - consul:consul
> >    volumes:
> >      - vault/assets/conf.d:/vault/conf.d
> > +  environment:
> > +    - "SERVICE_NAME=vault"
> > +    - "SERVICE_TAGS=vault"
> > +    - "VAULT_ADDR=http://0.0.0.0:8200"
> > +    - "VAULT_TOKEN=lofasz"
>
> Really?? [image: :confused:]
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/Sweebr/terra-con-vault/pull/1/files#r43101276>.
>